treeherder/model/management/commands/import_reference_data.py

```diff
@@ -66,7 +66,10 @@ def handle(self, *args, **options):
 
         # job type
         for job_type in c.get_job_types():
-            jgroup = JobGroup.objects.get(id=job_type['job_group'])
+            if job_type['job_group']:
+                jgroup = JobGroup.objects.get(id=job_type['job_group'])
```

Comment:
You can skip this query since you are only executing it to swap the id for its object.
You can directly pass the id to the defaults:
```python
defaults={
    'job_group_id'=job_type['job_group'],
    ....
}
```